Presentation/Slides-day-1.Rmd

```diff
@@ -0,0 +1,534 @@
+glimpse(penguins)
+```
+
+---
+## Warm-up Exercises: WarmUpExercises.Rmd
+
+- <b>Let's open RStudio cloud and do some warm-up examples</b>
+
+- Explore the penguins data
+  - How many penguins of each species are there? <i>We will do this one together</i>
+  - How many penguins of each species and sex are there? Hint: use `count`
+
+
+---
+## Ex. 1: How many penguins of each species are there? 
+
+```{r peng1}
+penguins %>%
+   count(species)
+```
+
+???
+- Using `count` we see there are 3 different species and the number of penguins for each species
+---
+## Ex. 2: How many penguins of each species and sex are there?
+
+```{r peng2}
+penguins %>%
+   count(species, sex)
+```
+???
+- `count` can take more than one variable to get a cross-tabs between the two variables
+
+---
+class: inverse center middle
+# Survey Datasets
+---
+## American National Election Studies (ANES) 2016 
+
+- Pre and post election surveys
+
+- Fielded almost every 2 years since 1948
+
+- Topics include voter registration status, candidate preference, opinions on country and government, party and ideology affiliation, opinions on policy, news sources, and more
+
+- Collaboration of Stanford, University of Michigan - funding by the National Science Foundation
+
+- <b>Target Population</b>: US citizens, 18 and older living in US 
+
+- <b>Mode</b>: FTF with CASI and Web
+
+- <b>Sample Information</b>: Pseudo-strata and pseudo-cluster included for variance estimation
+
+https://electionstudies.org/
+
+???
+- We have subset the columns of this data and created derived variables, code in repository
+- Chose not to use 2020 data because it is still preliminary
+
+---
+class: inverse center middle
+# Categorical descriptive data analysis
+---
+## Overview of Survey Analysis using `srvyr` Package
+
+1. Create a `tbl_svy` object using: `as_survey_design` or `as_survey_rep`
+
+2. Subset data (if needed) using `filter` (subpopulations)
+
+3. Specify domains of analysis using `group_by` 
+
+4. Within `summarize`, specify variables to calculate including means, totals, proportions, quantiles and more
+
+
+<b>Note: We will be teaching this in the reverse order!!!</b>
+---
+## Weighted Analysis for Categorical Variable
+
+- Functions to use within `summarize` after `group_by`
+   - survey_mean
+   - survey_total
+
+- Functions to get counts
+  - survey_count
+
+???
+
+- we use the same mean and total functions as with continuous variables
+- `survey_count` is new
+    - has a similar structure as the standard (non-survey) version of count
+
+
+---
+## Set-up for Analysis
+- `srvyr` package uses tidy-syntax but uses the `survey` package behind it to do calculations
+
+- If using your own RStudio environment, install both packages:
+```{r inst_srv, eval=FALSE}
+# Install survey and srvyr packages
+
+remotes::install_github("bschneidr/survey", ref = "c217689")
+install.packages("srvyr")
+```
+
+
+- First, we will set-up a design object and later talk about what it means
```

Comment:
"later" is really class 3.  Should we mention that in the slides, or just in talking about it.